packages/atlas-meta-server/src/api/index.ts

```diff
@@ -0,0 +1,23 @@
+import { GraphQLClient } from 'graphql-request'
+
+import { getSdk } from './__generated__/sdk'
+
+const { GRAPHQL_URL } = process.env
```

Comment:
I cannot comment `docker-compose` here but I think we should add `GRAPHQL_URL` also to some env file for development purposes.